File: 2028.py
# // 2028. Find Missing Observations
from typing import List

class Solution:
    def missingRolls(self, rolls: List[int], mean: int, n: int) -> List[int]:
        sum_rolls = sum(rolls)

        rem = mean * (n + len(rolls)) - sum_rolls

        if rem > 6 * n or rem < n:
            return []

        dist = rem // n
        mod = rem % n

        result = [dist] * n

        for i in range(mod):
            result[i] += 1

        return result

        

File: 2028.cpp
// 2028. Find Missing Observations
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> missingRolls(vector<int>& rolls, int mean, int n) {
        int m = rolls.size();
        vector<int> result;
        
        int sumOfRolls = 0;
        for (auto i: rolls){
            sumOfRolls += i;
        }

        int rem = mean * (m + n) - sumOfRolls;
        for (int i = 0; i < n; i++){
            
        }
        
    }
};

int main(){

}